asserts/aspect_bundle.go

```diff
@@ -106,3 +106,18 @@ func assembleAspectBundle(assert assertionBase) (Assertion, error) {
 		timestamp:     timestamp,
 	}, nil
 }
+
+// GetLocalAspectBundle returns a locally stored aspect bundle identified by
+// the account and bundle name.
+func GetLocalAspectBundle(db RODatabase, account, bundleName string) (*aspects.Bundle, error) {
```

Comment:
this should just be AspectBundle. For the doc comment follow the pattern of something like SnapDeclaration

Reply:
sorry, this is the wrong place =>  I wrote I think assertstate